crm_security_group/security/ir.model.access.csv

```diff
@@ -0,0 +1,6 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_crm_lead_user,crm.lead.user,crm.model_crm_lead,crm_security_group.group_crm_own_leads,1,1,1,0
+access_crm_team_user,crm.team.user,sales_team.model_crm_team,crm_security_group.group_crm_own_leads,1,0,0,0
+access_crm_team_manager,crm.team.manager,sales_team.model_crm_team,crm_security_group.group_crm_own_leads,1,1,1,1
+access_crm_stage_manager,crm.stage.manager,crm.model_crm_stage,crm_security_group.group_crm_manager,1,1,1,1
+access_crm_phonecall_security_group,crm.phonecall.user,crm_phonecall.model_crm_phonecall,crm_security_group.group_crm_own_leads,1,1,1,0
```

Comment:
crm_phonecall is not in the dependency of the module. So installing crm_security_group in a fresh database fail.